content/applications/inventory_and_mrp/inventory/management/delivery/dropshipping.rst

```diff
@@ -1,109 +1,80 @@
+=================================
 
-Create a *Sales Order* and add the product on which you just set the
-vendor. Add the *Route* field thanks to the widget on the right of the
-sale order line. Now, you are able to specify that your products’ route
-is *Drop-Shipping*.
+When a customer completes an online purchase for a dropshipped product, a sales order will be
+automatically created with an associated request for quotation (:abbr:`RFQ (Request for Quotation)`)
+for the supplier. Sales orders can be viewed in the :guilabel:`Sales` app by selecting
+:menuselection:`Orders --> Orders`. Click the :guilabel:`Purchase` smart button at the top right of
+a sales order to view the associated :abbr:`RFQ (Request for Quotation)`.
 
-.. image:: dropshipping/dropshipping_04.png
-    :align: center
+.. image:: dropshipping/dropship-sales-order.png
+   :align: center
+   :alt: A dropship sales order with the Purchase smart button in the top right corner.
 
-.. image:: dropshipping/dropshipping_05.png
-    :align: center
+.. tip::
+   Dropship sales orders can also be created manually through the :guilabel:`Sales` app by selecting
+   :menuselection:`Orders --> Orders --> Create` and adding a product configured for dropshipping.
 
-Once the sale order is confirmed, Odoo automatically generates a
-*Request for Quotation* for the supplier who will proceed to the
-drop-shipping. You can find it in the *Purchase* app, with the sale
-order as *Source Document*.
+Once the :abbr:`RFQ (Request for Quotation)` is confirmed, it becomes a purchase order and a
+dropship receipt is created and linked to it. The receipt can be viewed by clicking the
+:guilabel:`Receipt` smart button at the top right of the purchase order.
 
-.. image:: dropshipping/dropshipping_06.png
-    :align: center
+.. image:: dropshipping/dropship-purchase-order.png
+   :align: center
+   :alt: A dropship purchase order with the Receipt smart button in the top right corner.
 
-Once this *Purchase Order* is confirmed, a *Receipt Order* is
-created and linked to it. The source location is the vendor location and
-the destination location is the customer location. Then, the product
-won’t go through your own stock when you validate the dropship document.
+The dropship receipt will list the supplier in the :guilabel:`Receive From` section and the customer
+in the :guilabel:`Destination Location` section. Upon delivery of the product to the customer, click
+on the :guilabel:`Validate` button at the top left of the dropship receipt to confirm the delivered
+quantity.
 
-.. image:: dropshipping/dropshipping_07.png
-    :align: center
+.. image:: dropshipping/validate-dropship-receipt.png
+   :align: center
+   :alt: Validate the dropship receipt after delivery.
 
-You can also easily access the dropship document directly from your
-inventory dashboard.
+To view all dropship orders, simply navigate to the :guilabel:`Inventory` dashboard and click the
+green button on the :guilabel:`Dropship` card.
```

Comment:
I've seen it written like this on other docs.
Also teal > green (since it's a main Odoo color) 😉 
```suggestion
teal :guilabel:`# TO PROCESS` button on the :guilabel:`Dropship` card.
```